lib/chef/knife/status.rb

```diff
@@ -68,12 +72,21 @@ def run
         append_to_query("chef_environment:#{config[:environment]}") if config[:environment]
 
         if config[:hide_healthy]
+          ui.warn("-H / --hide-healthy is deprecated. Use --hide-by-mins MINS instead")
```

Comment:
I'm sure there's value. I asked about a code style guide re: deprecation (at least) in `#chef-hacking` the other day and was informed that I'd have to bring it up during developer hours. Currently deprecation seems like it takes many flavors (based on grepping around).

I wasn't aware of `Chef::Log.deprecation`. I saw a `ui.warn` deprecation usage in some other knife code and rolled with that.

This seems like maybe something for a deprecation "get right" effort though, after there's style documentation on it so that it doesn't keep drifting.